src/modules/content-tree/content.tree.module.js

```diff
@@ -149,6 +148,55 @@ export default class ContentTreeModule extends Component {
         );
     }
 
+    expandCurrentLocationInSubtree() {
+        const { rootLocationId, currentLocationPath } = this.props;
+        const path = currentLocationPath.split('/').filter((id) => !!id);
+        const rootLocationIdIndex = path.findIndex((element) => parseInt(element, 10) === rootLocationId);
+
+        if (rootLocationIdIndex === -1) {
+            return;
+        }
+
+        const pathStartingAfterRootLocation = path.slice(rootLocationIdIndex - path.length + 1);
+
+        this.expandPathInSubtree(this.subtree[0], pathStartingAfterRootLocation);
+    }
+
+    expandPathInSubtree(subtree, path) {
+        if (!path.length) {
+            return;
+        }
+
+        const locationId = parseInt(path[0], 10);
+        let nextSubtree = subtree.children.find((element) => element.locationId === locationId);
+
+        if (!nextSubtree) {
+            nextSubtree = {
+                '_media-type': 'application/vnd.ez.api.ContentTreeLoadSubtreeRequestNode',
+                locationId: locationId,
+                limit: 1,
```

Comment:
This probably will have to be resolved by backend because in the frontend I don't know at what position is e.g. currentLocation which I have to expand automatically.

Because of this this PR is WIP now.